packages/logging/src/index.js

```diff
@@ -444,7 +479,46 @@ Logging.prototype.getSinks = function(options, callback) {
  *     this.end();
  *   });
  */
-Logging.prototype.getSinksStream = common.paginator.streamify('getSinks');
+Logging.prototype.getSinksStream = function(options) {
```

Comment:
Is it worth making some kind of `requestStream_` helper method here? Seems like a lot of the code between streaming methods is similar.